kobalt/src/Build.kt

```diff
@@ -4,20 +4,19 @@ import com.beust.kobalt.plugin.publish.bintray
 import com.beust.kobalt.project
 import com.beust.kobalt.repos
 
-val r = repos("https://dl.bintray.com/cbeust/maven")
-
-//val pl = plugins(file(homeDir("kotlin/kobalt-retrolambda/kobaltBuild/libs/kobalt-retrolambda-0.3.jar")))
+val bs = buildScript {
+    val r = repos("https://dl.bintray.com/cbeust/maven")
+    //val pl = plugins(file(homeDir("kotlin/kobalt-retrolambda/kobaltBuild/libs/kobalt-retrolambda-0.3.jar")))
+}
 
 val p = project {
     name = "kobalt-retrolambda"
     artifactId = name
     group = "com.beust"
-    version = "0.9"
+    version = ".939"
```

Comment:
Looks good. I'll update a more up to date plugin api since that one is still 1.0.6.